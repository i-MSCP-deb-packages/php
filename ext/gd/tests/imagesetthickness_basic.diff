001+ 35159e543bf0a2ed0741bc26e62e80ad
001- 93c3077f1bdc372cd0b0db96db282985