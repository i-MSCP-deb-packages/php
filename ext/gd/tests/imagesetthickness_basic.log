
---- EXPECTED OUTPUT
93c3077f1bdc372cd0b0db96db282985
---- ACTUAL OUTPUT
35159e543bf0a2ed0741bc26e62e80ad
---- FAILED
