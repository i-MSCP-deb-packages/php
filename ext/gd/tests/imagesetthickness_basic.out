35159e543bf0a2ed0741bc26e62e80ad